technologiestiftung/giessdenkiez-de
# feat(Filter): implements new filter for water need #205

> This PR adds the functionality to filter by the need of water.

## Demo

https://user-images.githubusercontent.com/1086961/124952034-31899500-e014-11eb-8645-7de9495ca1c1.mp4

## Steps to review
1. `npm start`
2. Open search
3. Choose "**Niedrig 💧**" as Wasserbedarf filter
4. Confirm that trees have been filtered
5. Choose "**Mittel 💧**" as Wasserbedarf filter
6. Confirm that trees have been filtered
7. Choose "**Hoch 💧**" as Wasserbedarf filter
8. Confirm that trees have been filtered
9. Click "**Hoch 💧**" filter again to deselect it
10. Confirm that trees are not filtered anymore
